Mage.Tests/src/test/java/org/mage/test/cards/single/clb/GaleWaterdeepProdigyTest.java

```diff
@@ -0,0 +1,49 @@
+package org.mage.test.cards.single.clb;
+
+import mage.constants.PhaseStep;
+import mage.constants.Zone;
+import org.junit.Test;
+import org.mage.test.serverside.base.CardTestPlayerBase;
+
+/**
+ *
+ * @author R
+ * jayz
+ */
+public class GaleWaterdeepProdigyTest extends CardTestPlayerBase {
+
+    @Test
+    public void TestGaleWaterDeepProdigy() {
+        addCard(Zone.BATTLEFIELD, playerA, "Island", 3);
+        addCard(Zone.BATTLEFIELD, playerA, "Mountain", 3);
+
+        // Whenever you cast an instant or sorcery spell from your hand, you may cast up to one of the other type from your graveyard.
+        // If a spell cast from your graveyard this way would be put into your graveyard, exile it instead.
+        addCard(Zone.BATTLEFIELD, playerA, "Gale, Waterdeep Prodigy", 1);
+
+        // Draw two cards on a sorcery
+        addCard(Zone.HAND, playerA, "Divination");
+        addCard(Zone.GRAVEYARD, playerA, "Lightning Bolt");
+
+        playerA.getLibrary().clear();
+        addCard(Zone.LIBRARY, playerA, "Island");
+        addCard(Zone.LIBRARY, playerA, "Island");
+        skipInitShuffling();
+
+        setStrictChooseMode(true);
+
+        castSpell(1, PhaseStep.PRECOMBAT_MAIN, playerA, "Divination");
+        // Target Lightning Bolt in graveyard with Gale's ability
+        addTarget(playerA, "Lightning Bolt");
+
+        execute();
+
+        // Divination will have been cast, Lightning bolt should not have been cast
```

Comment:
Why should Lightning bolt not have been cast?

Reply:
I was going for a similar approach as for your advice on the test I wrote for [[Neera, Wild Mage]], but in hindsight it makes the test less clear. I have changed the test to hopefully be more clear.